_includes/main-title.html

```diff
@@ -27,4 +27,4 @@ <h4>resources for the community</h4>
       <img class="d-none d-md-block" src="assets/img/UX/logos/rocket.svg" align="right" style="max-height: 267px" />
     </div>
   </div>
-</div>
+</div>
```

Comment:
Also note that if you `{% include nav.html %}` without the `between-content` it will not try to push the cta in line with the nav and logo.

This is the kind of domain-knowledge I was trying to avoid but it's not _that_ bad